foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -49,7 +49,7 @@
 MAX_NUM_RETRIES = 2
 
 # This can be overritten by the env var "MAX_TOTAL_JOBS"
-DEFAULT_MAX_JOBS = 20000
+DEFAULT_MAX_JOBS = 5000
```

Comment:
Are there specific reasons why we are setting `5000` here? should we add a comment for it?

Reply:
Good point!